libmscore/timesig.cpp

```diff
@@ -299,6 +299,27 @@ void TimeSig::layout()
             ds.clear();
             }
       else {
+            // "other" timesigs are placed at the middle of the staff too.
+            const std::vector<SymId> prolatioIDList = {
```

Comment:
I don't like a vector with the same contet to be in the code twice, here and in mscore/timesigproperties.cpp
I guess the latter should bettere use this here (which means it needs to get externalized)